public/main.js

```diff
@@ -53,6 +58,14 @@ function reloadTodoList() {
             listItem.appendChild(completeButton);
             todoList.appendChild(listItem);
         });
+        if (count === 0) {
+            completeCounter.textContent = "";
+            completeCounter.className = "";
```

Comment:
Is there a reason for removing the class name? Would it make more sense to just use css to hide the element when the count is 0?